hash/msx1_bee_card.xml

```diff
@@ -0,0 +1,153 @@
+		<description>Star Force (Japan)</description>
+		<year>1985</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="serial" value="BC-M2" />
+		<info name="alt_title" value="スターフォース" />
+		<part name="cart" interface="bee_card">
+			<dataarea name="rom" size="32768">
+				<rom name="star force (japan).rom" size="32768" crc="c14e53a1" sha1="c1746efeb4b44dcc80dfc4f429009feac91d9c48" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="starfrcea" cloneof="starfrce">
+		<description>Star Force (Japan, alt)</description>
+		<year>1985</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="serial" value="BC-M2" />
+		<info name="alt_title" value="スターフォース" />
+		<part name="cart" interface="bee_card">
+			<dataarea name="rom" size="32768">
+				<rom name="star force (japan) (alt 1).rom" size="32768" crc="e9de7e32" sha1="5fe86465119343804a722ce4768eb33de806a39f" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="starsold">
+		<description>Star Soldier (Japan)</description>
+		<year>1986</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="serial" value="BC-M8" />
+		<info name="alt_title" value="スターソルジャー" />
+		<part name="cart" interface="bee_card">
+			<dataarea name="rom" size="32768">
+				<rom name="star soldier (japan).rom" size="32768" crc="0b3d975d" sha1="6a56b46b8bf014b25863433d6d96e64641a93eef" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="starsolda" cloneof="starsold">
+		<description>Star Soldier (Japan, alt)</description>
+		<year>1986</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="serial" value="BC-M8" />
+		<info name="alt_title" value="スターソルジャー" />
+		<part name="cart" interface="bee_card">
+			<dataarea name="rom" size="32768">
+				<rom name="star soldier (japan) (alt 1).rom" size="32768" crc="f6d4e101" sha1="b58caf47c953b6f00935a201bfccf7ee968dff32" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="takameij">
+		<description>Takahashi Meijin no Boukenjima (Japan)</description>
+		<year>1986</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="serial" value="BC-M9" />
+		<info name="alt_title" value="高橋名人の冒険島" />
+		<part name="cart" interface="bee_card">
+			<dataarea name="rom" size="32768">
+				<rom name="wonder boy (japan).rom" size="32768" crc="892266ca" sha1="d74c381ea17acb94527ef3abaa280793a824d322" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="takameija" cloneof="takameij">
+		<description>Takahashi Meijin no Boukenjima (Japan, alt)</description>
+		<year>1986</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="serial" value="BC-M9" />
+		<info name="alt_title" value="高橋名人の冒険島" />
+		<part name="cart" interface="bee_card">
+			<dataarea name="rom" size="32768">
+				<rom name="wonder boy (japan) (alt 1).rom" size="32768" crc="5130c27b" sha1="1672af5bc1740480e8302f7690589ea5b5391bcb" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="yakyukyo">
+		<description>Yakyu Kyo (Japan)</description>
```

Comment:
This should probably be romanized as "Yakyuu Kyou". That will bring it in line with most of the other softlists that have a port of this Hudson title.

Reply:
Updated the title.